### DISCUSSION TITLE
Downloadable Documentation For Offline Reading?

### DISCUSSION THREAD

**Discussion by muellert** [fastapi/fastapi#11748] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
# Code is not applicable, this is a process question.
```

### Description

I am in an ill-connected environment and would like to have downloadable documentation, like eg. available on readthedocs.io, so I can download and view it locally. My current connection is such that I can't read the docs and work on the code at the same time. I need to change the network configuration in order to switch between reading the docs and working (and trying out) the code. This is very time consuming and fraught with error, as my Internet connection keeps breaking down every few minutes, too.

### Operating System

Linux

### FastAPI Version

0.111.0

### Pydantic Version

2.7.4

### Python Version

3.11.2

**Comment by JavierSanchezCastro** [comment, marked as answer]
Check this: https://fastapi.tiangolo.com/contributing/#docs

**Reply by muellert** [reply]
Thank you!
